apps/web/pages/orgs/[orgSlug]/members/index.tsx

```diff
@@ -0,0 +1,9 @@
+import PageWrapper from "@components/PageWrapper";
```

Comment:
Based on our post in Campsite, I think we decided to actually move this to /settings/organizations/[id]/members to match the other routes

Reply:
Oh I see. I misunderstood it then. ([ref](https://linear.app/calcom/issue/CAL-4675/move-organization-settingsorganizationsmembers-to-members#comment-88cbfa9d))

Working on it now